julia/deps/build.jl

```diff
@@ -137,7 +138,7 @@ if !libmxnet_detected
 
     run(download_cmd(package_url, "mxnet.7z"))
     # this command will create the dir "usr\\lib"
-    run(`$exe7z e mxnet.7z *\\build\\* *\\lib\\* -y -ousr\\lib`)
+    run(`$exe7z e mxnet.7z "*\\build\\*" "*\\lib\\*" -y -ousr\\lib`) # TODO check it works on windows or not
```

Comment:
Does this imply we "might" break cmake build for windows? Can't we check this on windows before merging PR?

Reply:
ah, I checked it on Window, but forgot to remove the comment.
The previous code just poped  some warning about quoting.